addons/packages/secretgen-controller/0.5.0/bundle/config/values.yaml

```diff
@@ -0,0 +1,7 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
+---
+namespace: secretgen-controller
+secretgenController:
+  namespace: null
```

Comment:
Why do we have 2 variables for namespace? Probably just need one with default being `secretgen-controller`

Reply:
Fixed